deb-get

```diff
@@ -922,6 +922,14 @@ function deb_exodus() {
     WEBSITE="https://exodus.com/"
 }
 
+function deb_jabref() {
+    ARCHS_SUPPORTED="amd64"
+    get_github_releases "https://api.github.com/repos/jabref/jabref/releases/latest"
+    VERSION_PUBLISHED="$(grep "browser_download_url.*.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4 | cut -d'/' -f8 | sed 's|v||')"
+    URL=$(grep "browser_download_url.*.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)
+    PRETTY_NAME="GitHub Desktop"
```

Comment:
Please correct the PRETTY_NAME and WEBSITE

Reply:
Right!
I started changing it with the github data, and removed the wrong lines. It should be fixed now